lighthouse-core/audits/non-composited-animations.js

```diff
@@ -19,21 +19,25 @@ const UIStrings = {
   =1 {# animated element found}
   other {# animated elements found}
   }`,
-  /** Name of a compositor failure reason where the CSS property being animated is not supported on the compositor. */
-  unsupportedCSSProperty: 'Unsupported CSS Property',
+  /**
+   * @description Name of a compositor failure reason where the CSS property being animated is not supported on the compositor.
+   * @example {height, width} properties
+   */
+  unsupportedCSSProperty: 'Unsupported CSS Properties: {properties}',
 };
 
 const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
 /**
  * Each failure reason is represented by a bit flag. The bit shift operator '<<' is used to define which bit corresponds to each failure reason.
  * https://source.chromium.org/search?q=f:compositor_animations.h%20%22enum%20FailureReason%22
- * @type {{flag: number, text: string}[]}
+ * @type {{flag: number, text: string, needsUnsupportedProperties?: boolean}[]}
  */
 const ACTIONABLE_FAILURE_REASONS = [
   {
     flag: 1 << 13,
-    text: str_(UIStrings.unsupportedCSSProperty),
+    text: UIStrings.unsupportedCSSProperty,
+    needsUnsupportedProperties: true,
```

Comment:
This felt a bit weird since this failure reason is the only one that will use the unsupported property names.

I think it's good to keep all the failure reasons grouped together but any thoughts on completely separating this failure reason into a different structure?

Reply:
you could change `ACTIONABLE_FAILURE_REASONS` to be an object, and below do a check like `if (reason === ACTIONABLE_FAILURE_REASONS.UnsupportedProperty) ...`